### PR TITLE
Updated git pull index.md (minor changes)

### PR DESCRIPTION
**Pull request by dsaw**
* fixed typos
* removed repetitive content in git pull

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Thank you for opening this pull request.

This is a standard message notifying you that we’ve reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

Thank you and happy coding.

File: guide/english/git/git-pull/index.md [hunk 1]
```diff
@@ -70,27 +69,15 @@ The four network commands are:
...
-Without any arguments, `git merge` will merge the corresponding remote tracking branch to the local working branch. 
-
-#### git pull
-`git fetch` updates remote tracking branches. `git merge` updates the current branch with the corresponding remote tracking branch. Using `git pull`, you get both parts of these updates. But, this means that if you are checked out to `feature` branch and you execute `git pull`, when you checkout to `master`, any new updates will not be included. Whenever you checkout to another branch that may have new changes, it's always a good idea to execute `git pull`. 
 
 ### git pull in IDEs
 Common language in other IDES may not include the word `pull`. If you look out for the words `git pull` but don't see them, look for the word `sync` instead. 
 
-### fethcing a remote PR (Pull Request) in to local repo
+### fetching a remote PR (Pull Request) in to local repo
```
Comment (thecodingaviator) [inline guide/english/git/git-pull/index.md]:
```suggestion
### Fetching a remote PR (Pull Request) in to local repo
```
Correct the capitalization

**Comment by RandellDawson** [comment]
@thecodingaviator This one has gone stale.

**Comment by thecodingaviator** [comment]
Hi @dsaw ,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**